packages/flutter/lib/src/material/back_button.dart

```diff
@@ -84,7 +87,11 @@ class BackButton extends StatelessWidget {
       color: color,
       tooltip: 'Back', // TODO(ianh): Figure out how to localize this string
       onPressed: () {
-        Navigator.of(context).maybePop();
+        Navigator.of(context).maybePop().then((bool didPop) {
+          if (didPop)
+            return;
+          SystemNavigator.pop();
+        }).catchError((dynamic error) {});
```

Comment:
Idiomatic Dart would use `_` instead of `dynamic error` to indicate that you're not doing anything with the error, but the Flutter-specific lints probably don't allow that.